tests/pipelines/lint/test_files_exist.py

```diff
@@ -62,7 +62,7 @@ def test_files_exist_pass_conditional(self):
         lib_dir.mkdir()
         (lib_dir / "nfcore_external_java_deps.jar").touch()
         results = lint_obj.files_exist()
-        assert results["failed"] == []
+        assert results["failed"] == ["File must be removed: `lib/nfcore_external_java_deps.jar`"]
```

Comment:
This test was to test files that shouldn't exist if a condition is met, since we don't have such files anymore, i think we can remove the whole test.